bin/cli.js

```diff
@@ -149,6 +151,17 @@ const setRequestHeaders = () => {
 };
 
 const loadConfigMiddleware = async options => {
+  const validateConfig = () => {
```

Comment:
If you name this something like `maybeValidateConfig` then you wouldn't need to rename the import for `validateConfig`. It gets a little confusing when we use the exact same name for functions.

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1307/commits/24c78e289fc7bf4e8832826b131a7ce821d8c462